index.html

```diff
@@ -249,14 +249,34 @@ <h1>The Registration Process</h1>
 Any method name SHOULD avoid generic terms such as "mymethod" or "registry".
       </li>
       <li>
-If there are copyright, trademark, or any intellectual property rights
-concerns, the addition and use MUST be authorized in writing by the intellectual
-property rights holder under a
-<a href="https://en.wikipedia.org/wiki/Reasonable_and_non-discriminatory_licensing">F/RAND</a>
-license. Examples include DID Methods that use trademarked brand names,
-property names that utilize the titles of copyrighted works, and patented
-technology that would cause the use of the extension to require licensing a
-patent.
+Any DID Method submission MUST not impose any copyright, trademark, or 
+intellectual property rights concerns on the W3C, DID Method implementers, or 
+users.  To that end, the following apply:
+          <ol>
+          <li>
+DID Method submitters affirm to the best of their knowledge that their 
+submission will not infringe upon or otherwise violate the rights of any third 
+party.
+          </li>
+          <li>
+If there are any known copyright, trademark, or intellectual property rights 
+concerns, the addition and use of a DID Method MUST be authorized in writing 
+by the intellectual property rights holder under a 
+<a href="https://en.wikipedia.org/wiki/Reasonable_and_non-discriminatory_licensing">F/RAND</a> 
+license.
+          </li>
+          <li>
+If a DID Method is subsequently alleged to violate any copyright, trademark, 
+or intellectual property rights, those making the allegation SHALL provide a 
+copy of their granted legal rights documentation or an order from a court or 
+arbitral tribunal, in each case of competent jurisdiction.
```

Comment:
Wondering what the process on the maintainers end should be in this case. Do we just leave the method being challenged in the list unchanged while the challenge is ongoing? Or flag it in someway?

Reply:
Also wondering about the edge case where there are two trademarks for the same mark valid in distinct jurisdictions.

Reply:
Great questions.  It might be good to note a trademark infringement allegation somewhere.  However, an allegation shouldn't change anything operationally until there is official documentation of ownership.  

The multiple jurisdictions questions is a more complicated question.  In that event, we might just have 2 valid, yet separate entries.  For example, a US patent only governs implementations the US, while those in the EU may freely implement it.  

My intent with this PR was to adopt ICANN's process, which essentially allows domain names to be registered until someone can legally prove that their trademark, copyright, or IP is being violated.